src/table/cells/handle-cell-focus.js

```diff
@@ -80,3 +80,49 @@ export const handleNavigateTop = ({ tableSection, focusedCellCoord, rootElement
     }
   }
 };
+
+const getMemoisedSrNotation = () => {
+  let prevSelectedCount = 0;
+
+  return ({ focusedCellCoord, rootElement, selState }) => {
+    if (focusedCellCoord[0] === 0) return '';
+
+    const [rowIdx, colIdx] = focusedCellCoord;
+    const rowElements = rootElement.getElementsByClassName('sn-table-row');
+    const cell = rowElements[rowIdx]?.getElementsByClassName('sn-table-cell')[colIdx];
+
+    const tCellContent = cell && cell.innerText;
+    const isCellSelected = cell && cell.classList.contains('selected');
+
+    let notation = '';
+
+    if (selState.rows.length) {
+      const isSingularSelection = selState.rows.length === 1;
+      const selectionNote = `There ${isSingularSelection ? 'is' : 'are'} ${selState.rows.length} selected value${
+        isSingularSelection ? '' : 's'
+      } currently.`;
```

Comment:
We need to consider how to solve this with strings. There is a way sending in a variable to a string, and then it places it at a specified place in the string. In this case the # of values. Also, this way of patching a string together, depending if there is one or multiple values selected wont work with strings. We could have just two different strings tho like this:
`There is <1> selected value currently`
`There are <2> selected values currently `

Reply:
Just need to make sure we can add arguments to strings

Reply:
I will separate it into a function of `getCellSelectionStatusNote`.

speaking of ( function ) template literals, i think we could handle it with just a simple function. and remove the redundant code from return function of `getMemoisedSrNotation` as well

Reply:
✅ Fixed

Reply:
Sure, would be nice to decrease the the complexity of this function. But all it will be later is something like this:
`selectionNote = translator(isSingularSelection ? 'table.singleSelection' : 'table.multipleSelection', selState.rows.length);`

Reply:
To clarify, we can't do this before Li has merged the translation PR. The way it is now is fine :)